rules/Proj0010.md

```diff
@@ -0,0 +1,26 @@
+# Proj0010: Define the OutputType explicitly
```

Comment:
output type are two words. (it is the term referred to, not the `<OutputType>` node.